.travis.yml

```diff
@@ -2,5 +2,4 @@ services:
   - docker
 
 script:
-  - docker-compose build
-  - docker-compose run --rm core
\ No newline at end of file
+  - ./test --watch
```

Comment:
doesn't look like you added anything related to `--watch` in the bash script
`--watch` is for the purpose of local development.

Reply:
So just to clarify this, without the `--watch` flag,  the entrypoint would be `crystal spec --error-trace`, but with the `--watch` flag, it would be instead `watchexec -c -e cr -w src -w spec -- crystal spec --error-trace -v`?

Reply:
yep